t.py

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
`black`: this file needs to be formatted  <!-- Automatically posted by Lintly -->